tools/custom-action/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash --login
+set -eo pipefail
+
+. /prepare.sh # Execute the prepare script
```

Comment:
nit: I prefer `source` over `.` for readability

Reply:
`. /prepare.sh` looks too similar to
`./prepare.sh`

Reply:
I also like source too but did this because of sh compatibility on reference. I will use source

Reply:
Done, thanks a lot!